### PR TITLE
CFE-4345: Added freebsd package_module and package_inventory since we have packages module available (3.18)

### PR DESCRIPTION
**Pull request by craigcomstock**
- Added ability to influence default package manager and inventory via Augments
- Added freebsd package_module and package_inventory since we have pkg packages module available

### CODE REVIEW

File: MPF.md [hunk 1]
```diff
@@ -1735,6 +1735,53 @@ For example:
...
+      },
+        "comment": "This variable provides the ability to override the default package manager to use for a platform. Keys are based on the value of $(sys.flavor) for the targeted platform."
+    }
+  }
+}
+```
+
+**History:**
+
+* Added in CFEngine 3.24.0
```
Comment (nickanderson) [inline MPF.md:1760]:
```suggestion
* Added in CFEngine 3.24.0, 3.21.5, 3.18.8
```

**Review by nickanderson — APPROVED** [review]
We should update the history.

File: MPF.md [hunk 2]
```diff
@@ -1735,6 +1735,53 @@ For example:
...
+        },
+        "comment": "This variable provides the ability to extend the default package managers to inventory for a platform. Keys are based on the value of $(sys.flavor) for the targeted platform."
+      }
+  }
+}
+```
+
+**History:**
+
+* Added in CFEngine 3.24.0
```
Comment (craigcomstock) [inline MPF.md:1783]: @nickanderson would we not update this one also? And then update master with these version changes ;) so I will wait for your answer about this one and then re-do the changes from master down... (the version bits here)

Comment (nickanderson) [reply MPF.md:1783]: Yes, this one too. I guess I just missed it. Doesn't matter to me which way the things are picked from as long as they all match.

**Pull request merged**